iron-swipeable-pages.html

```diff
@@ -253,6 +261,10 @@
 
         // prepare the page for animation and add it to the list of pages to be animated
         _initPage: function(page, transition, left) {
+          if (!page) {
+            return;
+          }
+
```

Comment:
Hi @MeTaNoV, see https://github.com/MeTaNoV/iron-swipeable-pages/issues/16 I think it is necessary.